Makefile

```diff
@@ -364,7 +364,7 @@ GATEWAY_API_PACKAGE ?= sigs.k8s.io/gateway-api
 #       $(shell go list -m -f "{{.Version}}" $(GATEWAY_API_PACKAGE))
 #
 #       Related issue: https://github.com/Kong/kubernetes-ingress-controller/issues/2595
-GATEWAY_API_VERSION ?= 71fee1c2808fa19a5f19d952d155fc072cf9324c
+GATEWAY_API_VERSION ?= v0.5.0
```

Comment:
Please remove the whole comment above this line as it's no longer valid given the update to 0.5.0.

The only thing I'd leave is the mention that only branch names, tags, or full commit hashes are supported (with the link to `kustomize`'s doc)

Reply:
:+1: